### PR TITLE
Render Transaction from content-store

### PR DESCRIPTION
**Pull request by whoojemaflip**
To be deployed after https://github.com/alphagov/publisher/pull/582 and after a republish of all transactions has been run.

[Trello](https://trello.com/c/B0WRTEfM/674-migrate-transaction-format-to-rendered)

### CODE REVIEW

File: app/views/transaction/_additional_information_single.html.erb [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
     <%= raw transaction.more_information %>
   </div>
 <% end %>
 
-<% if presenter.what_you_need_to_know? %>
+<% if transaction.what_you_need_to_know.present? %>
   <div id="what-you-need-to-know">
     <h2><%= t 'formats.transaction.what_you_need_to_know' %></h2>
-    <%= render :partial => 'need_to_know', :locals => { :transaction => transaction } %>
+    <%= raw transaction.what_you_need_to_know %>
```
Comment (nickcolley) [inline app/views/transaction/_additional_information_single.html.erb]: Can we use `transaction.what_you_need_to_know.try(:html_safe)` rather than raw?

Comment (binaryberry) [inline app/views/transaction/_additional_information_single.html.erb]: Sure! Just out of curiosity, could you explain why it's better?

Comment (binaryberry) [inline app/views/transaction/_additional_information_single.html.erb]: Amended for all instances of raw in the PR!

**Comment by brenetic** [comment]
👍  when tests are green

**Comment by davidbasalla** [comment]
Added 'do not merge' tag so that this doesn't get merged prematurely (before the transaction related schemas and publisher PRs have been merged/deployed)

File: app/views/transaction/_additional_information_single.html.erb (hunk 1 above)
Comment (binaryberry) [inline app/views/transaction/_additional_information_single.html.erb]: @nickcolley - we'll be merging this soon, if you weren't happy with the changes let us know.

**Pull request merged**